code/modules/mob/living/simple_animal/hostile/terror_spiders/mother.dm

```diff
@@ -0,0 +1,50 @@
+
+// --------------------------------------------------------------------------------
+// ----------------- TERROR SPIDERS: T3 MOTHER OF TERROR --------------------------
+// --------------------------------------------------------------------------------
+// -------------: ROLE: living schmuck bait
+// -------------: AI: no special ai
+// -------------: SPECIAL: spawns an ungodly number of spiderlings when killed
+// -------------: TO FIGHT IT: don't! Just leave it alone! It is harmless by itself... but god help you if you aggro it.
+// -------------: CONCEPT: http://tvtropes.org/pmwiki/pmwiki.php/Main/MotherOfAThousandYoung
+// -------------: CONCEPT: http://tvtropes.org/pmwiki/pmwiki.php/Main/SchmuckBait
+// -------------: SPRITES FROM: FoS, http://nanotrasen.se/phpBB3/memberlist.php?mode=viewprofile&u=386
```

Comment:
Concept links are there to explain the design philosophy. I consider them necessary - when I look at this code in future I want myself and others to understand how they were designed, and why.

The fact that they lead to people wasting their lives on TVTropes is just an evil bonus.